libethereum/Executive.cpp

```diff
@@ -400,10 +400,14 @@ void Executive::finalize()
 	m_gas += m_refunded;
 
 	//	cnote << "Refunding" << formatBalance(m_endGas * m_ext->gasPrice) << "to origin (=" << m_endGas << "*" << formatBalance(m_ext->gasPrice) << ")";
-	m_s.addBalance(m_t.sender(), m_gas * m_t.gasPrice());
```

Comment:
Executive is created without a transaction from Client::call and this throws an exception